docs/warnings.rst

```diff
@@ -27,7 +27,7 @@ script using the `warnings.filterwarnings` function as follows::
      >>> import warnings
      >>> from astropy.io import fits
      >>> warnings.filterwarnings('ignore', category=UserWarning, append=True)
-     >>> fits.writeto(filename, data, clobber=True)
```

Comment:
Unless I'm mistaken, I think the point of these examples is to show how to suppress warnings from `astropy` if necessary. The `overwrite=True` flag is being used in the examples since it will cause a warning which could be suppressed.